Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -405,7 +439,8 @@ def collect_pipeline_data(gitlab_client: gitlab.Gitlab,
 def construct_coverage_slack_msg() -> list[dict[str, Any]]:
     coverage_today = get_total_coverage(filename=(ROOT_ARTIFACTS_FOLDER / "coverage_report" / "coverage-min.json").as_posix())
     coverage_yesterday = get_total_coverage(date=datetime.now() - timedelta(days=1))
-    coverage_last_month = get_total_coverage(date=datetime.now() - relativedelta(months=-1))
+    # The artifacts are kept for 30 days, so we can get the coverage for the last month.
+    coverage_last_month = get_total_coverage(date=datetime.now() - timedelta(days=30))
```

Comment:
had a bug: we keep the coverage for only 30 days, so using relativedelta might generate a 31 days diff, which caused the coverage reported to be `0%`